packages/blocks/src/page-block/edgeless/components/text/edgeless-text-editor.ts

```diff
@@ -63,10 +56,10 @@ export class EdgelessTextEditor extends WithDisposable(ShadowlessElement) {
       overflow-wrap: normal !important;
     }
 
-    /* We cannot add styles directly from the top, as this would cause a shift in the inline elements inside. */
-    /* https://github.com/toeverything/blocksuite/issues/5723 */
```

Comment:
This will cause a bug regression.

Reply:
Well, in an expanded text box... the editor text is always left aligned... 
to get proper alignment of canvas-text with editor and even the placeholder having top level text align works very well. This is what I intended to solve with this pr. 
Is there some specific bug with this implementation?

https://github.com/toeverything/blocksuite/assets/54364088/9352c360-32ab-4a90-b318-e02e39eac465

Reply:
finally understood the bug and problem
found a way to use top level textAlign an prevent excess shift in inline elements
textAlign right in edit mode:-
<img width="454" alt="image" src="https://github.com/toeverything/blocksuite/assets/54364088/854e4aa0-f9ba-452d-8029-eccde7b86fd1">